### ISSUE TITLE
Optionally generate wrist rests

### ISSUE THREAD

**Issue by properlypurple** [50an6xy06r6n/hotswap_pcb_generator#18]
I love this feature in [Klavgen](https://github.com/klavgen/klavgen/), where it generates wrist rests as well, either as separate parts, or integrated in the case. It would be really cool to have something like this as an optional part.

**Comment by 50an6xy06r6n** [comment]
Yeah that was a feature I was thinking about as well. Will put that on the list of potential features whenever I revisit this project
